.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: python
-python: 3.5
-install: 
+python: 3.6
+cache: pip
+install:
   - ./setup
   - pip install coveralls
-script: 
+script:
   - coverage run rosie.py test
-
+  - python rosie.py run federal_senate
```

Comment:
I do agree, @jtemporal — but I also see the point: better to run and make sure it doesn't crash than have nothing in the CI at all. Maybe both?